Userland/Libraries/LibGL/GLContext.h

```diff
@@ -543,6 +544,12 @@ class GLContext final {
 
     // GL Extension string
     String m_extensions;
+
+    // Buffer objects
+    NameAllocator m_buffer_name_allocator;
+    HashMap<GLuint, RefPtr<Buffer>> m_allocated_buffers;
+    Buffer* m_array_buffer;
+    Buffer* m_element_array_buffer;
```

Comment:
Can't we make these `RefPtr<Buffer>` as well?

Reply:
Ah, this was what was causing the tests to fail. The pointers weren't initialized so `glDrawElements` was assuming that a buffer was bound when that wasn't the case.